Formula/luaver.rb

```diff
@@ -4,13 +4,8 @@ class Luaver < Formula
   head "https://github.com/DhavalKapil/luaver.git"
 
   stable do
-    url "https://github.com/DhavalKapil/luaver/archive/v1.0.0.tar.gz"
-    sha256 "5c7eb65ea9b3cb4f10987c1d564e6e86bd6ad4f9d829a799121c83f0d85bd390"
-
-    patch do
-      url "https://github.com/DhavalKapil/luaver/pull/9.patch?full_index=1"
-      sha256 "0fa92083eabc2e9cb369c3aefbcf17cee15442ad8c22b18cbe6e2a39e226ca8e"
-    end
+    url "https://github.com/DhavalKapil/luaver/archive/v1.1.0.tar.gz"
```

Comment:
The url and checksum should be moved above head and the `stable do ... end` removed.